clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -3505,6 +3524,7 @@ public void testClientInstanceIdNoTelemetryReporterRegistered(GroupProtocol grou
         assertEquals("Telemetry is not enabled. Set config `enable.metrics.push` to `true`.", exception.getMessage());
     }
 
+    @SuppressWarnings("resource")
```

Comment:
It looks like this is redundant.

Reply:
IntelliJ warning. `newConsumer` returns an auto-closable